sympy/tensor/tests/test_tensor.py

```diff
@@ -1968,3 +1968,14 @@ def test_TensorType():
         Lorentz = TensorIndexType('Lorentz')
         S2 = TensorType([Lorentz]*2, sym2)
         assert isinstance(S2, TensorType)
+
+@filter_warnings_decorator
+def test_hooks():
```

Comment:
Why are there warnings coming from the new test?

Reply:
It's because `_get_valued_base_test_variables` raises deprecation warning. Every other tests which involves tensor's operation use that function, and they have this decorator. I don't know tensor module well so I just followed other tests blindly. Any suggestions?

Reply:
Perhaps out of scope for this PR but the proper solution is that `_get_valued_base_test_variables` which is part of the testing code here should not be using deprecated code.

Reply:
Please refer to #15276. Maybe we will need new `TensPow` class, but that needs to be done in another PR so I'll mark this resolved.